prisma/dev-seed-utils/applications.ts

```diff
@@ -298,6 +314,10 @@ const applications: Array<UpsertApplication> = [
       otherRequiresWiderParkingSpaceReason: null,
     },
     replacementApplication: undefined,
+    paidThroughShopify: true,
+    shopifyPaymentStatus: ShopifyPaymentStatus.RECEIVED,
```

Comment:
```suggestion
    shopifyPaymentStatus: "RECEIVED",
```
Let's use string value here for consistency